packages/react-composites/src/composites/common/Drawer/MoreDrawer.tsx

```diff
@@ -193,6 +193,7 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
   if (drawerSelectionOptions !== false && isEnabled(drawerSelectionOptions?.peopleButton)) {
     drawerMenuItems.push({
       itemKey: 'people',
+      id: 'call-composite-drawer-people-button',
```

Comment:
Should this be a data-ui-id? I think this property sets the DOM id we are trying to avoid.

Reply:
It should be ideally, but seems like contextual menu doesn't allow us to set data-ui-id, so it is a workaround for this issue